warehouse/ingest-core/src/test/java/datawave/ingest/mapreduce/TableConfigHelperFactoryTest.java

```diff
@@ -6,27 +6,44 @@
 import datawave.util.TableName;
 import org.apache.accumulo.core.client.Connector;
 import org.apache.accumulo.core.client.admin.TableOperations;
-import org.apache.accumulo.core.client.mock.MockInstance;
-import org.apache.accumulo.core.client.security.tokens.PasswordToken;
+import org.apache.accumulo.minicluster.MiniAccumuloCluster;
+import org.apache.accumulo.minicluster.MiniAccumuloConfig;
+import org.apache.commons.io.FileUtils;
 import org.apache.hadoop.conf.Configuration;
 import org.apache.log4j.Logger;
+import org.junit.AfterClass;
 import org.junit.Before;
+import org.junit.BeforeClass;
 import org.junit.Test;
 
 import static org.hamcrest.core.Is.is;
 import static org.hamcrest.core.IsNull.nullValue;
 import static org.junit.Assert.assertThat;
 
+import java.io.File;
+
+/**
+ * Test uses mini accumulo cluster. Files are stored in warehouse/ingest-core/target/mac/datawave.ingest.mapreduce.TableConfigHelperFactoryTest
+ */
 public class TableConfigHelperFactoryTest {
     private static final Logger logger = Logger.getLogger(TableConfigHelperFactoryTest.class);
+    private static MiniAccumuloCluster mac;
     
     private Configuration conf;
-    private MockInstance instance;
-    private Connector connector;
     private TableOperations tops;
     
     private static final String TEST_SHARD_TABLE_NAME = "testShard";
     
+    @BeforeClass
+    public static void startCluster() throws Exception {
+        File macDir = new File(System.getProperty("user.dir") + "/target/mac/" + TableConfigHelperFactoryTest.class.getName());
+        if (macDir.exists())
+            FileUtils.deleteDirectory(macDir);
+        macDir.mkdirs();
```

Comment:
Should we have a deleteOnExit call for this, or delete it in the shutdown method?

Reply:
The mini accumulo files are useful for debugging failures in the Test.  The directory is in target so can easily be removed on a "mvn clean".